pages/phases/getting-started/identifying-appropriate-positions.md

```diff
@@ -15,13 +15,15 @@ intro: To meet the goals of your hiring action, gather information to drive your
 
 ### Identifying a Classified Position Description
 
-Your position description (PD) should reflect adequate specialized experience to require the time and effort of SMEs. Rather than classifying a new PD, using a standardized PD is encouraged since you will be doing an in-depth job analysis workshop to link the PD to an accurate job announcement. The competencies identified through the job analysis workshop simply need to be rooted in the PD. <a href="https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/deo_handbook.pdf">Page 3-12 of the updated DE Handbook clarifies</a>:
+Your position description (PD) should reflect adequate specialized experience to require the time and effort of SMEs. Rather than classifying a new PD, we encourage using a standardized PD since your in-depth job analysis workshop will link the PD to an accurate job announcement. The competencies identified through the job analysis workshop simply need to be rooted in the PD. <a href="https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/deo_handbook.pdf">Page 3-12 of the updated DE Handbook clarifies</a>:
 
 <blockquote>"Conditions of employment should be supported in the position description and/or by job analysis."</blockquote>
 
 
 ### Identify Billets
 
-To make this type of hiring action effective, you should have approved headcount of five or more GS-12+ billets for the same technical role. Fewer billets may not provide the necessary return on SME time required for this process.
+Our research suggests that the typical cost to hire a new full time employee in industry is $4000-$5000. One pilot agency spent ~$13000 (in personnel costs) on their hiring action, which onboard four people, suggesting a cost per hire of $3250. Other pilots have spent between $2500 and $6900 per hire on their SMEQA pilots.
 
-To make the effort worthwhile, you can share your certificate with another office or agency with vacancies in the same position to share the certificate. This also allows the two groups to share SMEs throughout the action.
+In order to be cost efficient, you should have approved headcount of three or more GS-12+ billets for the same technical role; however, five or more is recommended to achieve cost efficiency. Fewer billets presents a higher risk of spending above the industry average, though your agency may accept that cost for certain roles.
```

Comment:
again, let's stay away from the word "cost" since FTE time is not typically viewed as a "cost" to your average HR specialist. just "in order to maximize the time your SMEs are spending...."

Reply:
i'm not sure about this at all anymore- what about the language we were going to add about pulling a ranking list to make this more efficient for just 2-3 hires? Ro approved that recently but I don't see that reflected here.

Reply:
I don't think we need ranking lists for it to work for 3 people, but we can add a note about them here.

The ranking list info is here: https://github.com/usds/SME-QA-Website/pull/234/files